src/wallet/rpcwallet.cpp

```diff
@@ -2397,6 +2399,40 @@ UniValue lockunspent(const JSONRPCRequest& request)
 
         COutPoint outpt(uint256S(txid), nOutput);
 
+        const bool isLocked = pwallet->IsLockedCoin(outpt.hash, outpt.n);
+
+        if (fUnlock)
```

Comment:
IMO yes, it can indicate that it was locked by other client as such it is not yours to use?